src/mocks/mockSongsData/mockSongsData.ts

```diff
@@ -1,7 +1,18 @@
+import { faker } from "@faker-js/faker";
+import { Factory } from "fishery";
 import { Song } from "../../store/contexts/types";
 
-export const mockSong: Song = {
-  artist: "Armthis&Lojelzy",
-  picture: "arm_loj.jpg",
-  title: "Lo que diga Lorena",
-};
+const numberOfSongs = 2;
+
+const songFactory = Factory.define<Song>(() => ({
+  artist: faker.internet.userName(),
+  picture: faker.image.imageUrl(),
+  title: faker.music.songName(),
+  album: faker.company.name(),
+  audio: faker.system.filePath(),
+  id: faker.datatype.uuid(),
+  time: `${faker.datatype.datetime()}`,
+}));
+
+export const mockSong = songFactory.build();
```

Comment:
Esto te va a dar siempre el mismo mock. Cread una función que cuando la llaméis os devuelva cada vez un mock distinto (lo mismo para mockListSong)